middlewares/validators/validator.helper.js

```diff
@@ -607,6 +607,91 @@ function accountTypeValidator(fieldLocation, fieldname, optional = true) {
     }
 }
 
+/**
+ * Validates that field must be an array of routes
+ * @param {"query" | "body" | "header" | "param"} fieldLocation the location where the field should be found 
+ * @param {string} fieldname name of the field that needs to be validated.
+ * @param {boolean} optional whether the field is optional or not.
+ */
+function routesValidator(fieldLocation, fieldname, optional = true) {
+    const routes = setProperValidationChainBuilder(fieldLocation, fieldname, "Invalid routes");
+
+    if (optional) {
+        return routes
+            .optional({
+                checkFalsy: true
+            })
+            .custom(routesArrayValidationHelper).withMessage("The value must be a route");
+    } else {
+        return routes
+            .exists()
+            .withMessage("The value being checked agains the enums must exist.")
```

Comment:
`.withMessage("routes must exist")`

Reply:
If I understand, this message essentially just is shown when the route parameter does not exist

Reply:
Oops, bad copy paste